fmriprep/workflows/epi.py

```diff
@@ -248,7 +374,7 @@ def apply_fs_transform(fs_2_t1_transform, bbreg_transform):
             (inputnode, ds_t1w, [(('name_source', _first), 'source_file')]),
             (inputnode, ds_t1w_mask,
              [(('name_source', _first), 'source_file')]),
-            (merge, ds_t1w, [('merged_file', 'in_file')]),
+            (outputnode, ds_t1w, [('epi_t1', 'in_file')]),
```

Comment:
This seems to be a null change. Why use `outputnode` as the source? Just to move the `DerivativesDataSink` to the bottom of the graph?